src/plugins/data_view_management/public/components/edit_index_pattern/relationships_table/relationships_table.tsx

```diff
@@ -139,12 +140,20 @@ export const RelationshipsTable = ({
     ] as SearchFilterConfig[],
   };
 
+  const { pageSize, onTableChange } = useEuiTablePersist<SavedObjectRelation>({
+    tableId: 'dataViewMgmtRelationships',
+    initialPageSize: 10,
+  });
```

Comment:
unrelated to this PR, but just 10? and no sorting allowed? file under: bad defaults